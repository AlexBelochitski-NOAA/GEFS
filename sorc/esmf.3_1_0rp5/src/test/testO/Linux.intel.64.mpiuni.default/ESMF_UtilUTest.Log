20190610 112230.571646 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112230.571898 INFO     PET0  Beginning Test, file ESMF_UtilUTest.F90, line          68
20190610 112230.571916 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112230.571935 INFO     PET0  Number of failed tests:           0
20190610 112230.571941 INFO     PET0  Ending Test, file ESMF_UtilUTest.F90, line          80
