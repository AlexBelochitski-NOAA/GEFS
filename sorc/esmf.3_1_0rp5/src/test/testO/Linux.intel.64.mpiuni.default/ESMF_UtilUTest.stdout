  Beginning Test, file ESMF_UtilUTest.F90, line          68
  NUMBER_OF_PROCESSORS           1
  Number of failed tests:           0
  Ending Test, file ESMF_UtilUTest.F90, line          80
